ext/websocket/02_websocketstream.js

```diff
@@ -176,7 +176,7 @@ class WebSocketStream {
                         create.rid,
                       );
 
-                      if (kind > 6) {
```

Comment:
Is there a test that covers this change? If not can you add it?

Reply:
WPT covers it but this was never hit. I actually found this case by accident with a bug in fastwebsockets.